### DISCUSSION TITLE
Integrating with Hubspot

### DISCUSSION THREAD

**Discussion by JoeMatkin** [swellstores/community#448] [Help] [answered]
Hello,

Swell states it integrates with Hubspot but I can't see it as an option to enable in the integrations tab and I also don't see any docs on setting it up.

Is there a native integration with Hubspot? If not how would you suggest syncing customers between Hubspot and Swell.

Thanks.

**Comment by ericingram** [comment, marked as answer]
Thanks for the tip @JoeMatkin. We paused development on a HubSpot integration earlier this year, won't likely be picked up until someone builds an app for it.

We'll remove it from the website for now
